examples/preprocessing/plot_power_transformer.py

```diff
@@ -0,0 +1,85 @@
+"""
+==========================================================
+Using PowerTransformer to apply the Box-Cox transformation
+==========================================================
+
+This example demonstrates the use of the Box-Cox transform
+through ``PowerTransformer`` to map data from various
+distributions to a Gaussian distribution.
+
+Box-Cox is useful as a transformation in modeling problems
+where homoscedasticity and normality are desired. Below
+are examples of Box-Cox applied to three distributions: a
+heavily left-skewed lognormal distribution, a left-skewed
+chi-squared distribution, and a right-skewed Weibull
+distribution.
+
+Note that when Box-Cox is applied to the lognormal distribution,
+the estimated parameter, lambda, takes on a value close to
+zero since Box-Cox is defined as the natural logarithm when
+lambda = 0. In all cases, the maximum likelihood estimated
+lambdas achieve very close results to the normal distribution.
+"""
+print(__doc__)
+
+# Author: Eric Chang <ericchang2017@u.northwestern.edu>
+# License: BSD 3 clause
+
+import numpy as np
+import matplotlib
+import matplotlib.pyplot as plt
+
+from sklearn.preprocessing import PowerTransformer
+
+N_SAMPLES = 3000
+SEED = 304
+
+pt = PowerTransformer(method='box-cox')
+rng = np.random.RandomState(SEED)
+size = (N_SAMPLES, 1)
+
+# lognormal distribution
+X_lognormal = rng.lognormal(size=size)
+X_lognormal_trans = pt.fit_transform(X_lognormal)
+lmbda_lognormal = round(pt.lambdas_[0], 2)
+
+# chi-squared distribution
+df = 3
+X_chisq = rng.chisquare(df=df, size=size)
+X_chisq_trans = pt.fit_transform(X_chisq)
+lmbda_chisq = round(pt.lambdas_[0], 2)
+
+# weibull distribution
+a = 50
+X_weibull = rng.weibull(a=a, size=size)
+X_weibull_trans = pt.fit_transform(X_weibull)
+lmbda_weibull = round(pt.lambdas_[0], 2)
+
+# plot
+params = {'font.size': 6, 'hist.bins': 150}
+matplotlib.rcParams.update(params)
+
+fig, axes = plt.subplots(nrows=2, ncols=3)
+ax0, ax2, ax4, ax1, ax3, ax5 = axes.flatten()
```

Comment:
I would make a ploting function and loop other the distributions and just mapping the right title

Reply:
To reduce code duplication, I would as well, but I think this is more explicit, which is better for an example

Reply:
Correction - with 6 distributions, it became necessary :)